### PR TITLE
CI: ignore some not implemented methods code coverage

### PR DESCRIPTION
**Pull request by sfc-gh-azhan**
CI: ignore some not implemented methods code coverage

### CODE REVIEW

File: src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py [hunk 1]
```diff
@@ -7,13 +7,13 @@
 
 class StringMethods:
     def casefold():
-        pass
+        pass  # pragma: no cover
```
Comment (sfc-gh-mvashishtha) [inline src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py:10]: Should we skip coverage for the entire classes instead, as [here](https://github.com/snowflakedb/snowpark-python/blob/903eee29597687ac64d4041c903c106d7a3e84a8/src/snowflake/snowpark/modin/plugin/docstrings/base.py#L388)?

Comment (sfc-gh-helmeleegy) [reply src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py:10]: +1. This is what we do for all other object types, not just base. So better (and easier) to stay consistent.

Comment (sfc-gh-azhan) [reply src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py]: Yes, we skipped all in tox.ini. PTAL

File: tox.ini [hunk 2]
```diff
@@ -1,16 +1,18 @@
 [coverage:report]
 skip_covered = False
 show_missing = True
-# TODO: SNOW-1453027 remove omit
+# TODO: SNOW-1453027 remove omit for config and core
 omit = */src/snowflake/snowpark/modin/config/*
        */src/snowflake/snowpark/modin/core/*
+       */src/snowflake/snowpark/modin/plugin/docstrings/*
```
Comment (sfc-gh-azhan) [inline tox.ini:7]: @sfc-gh-helmeleegy we can just skip it here.

Comment (sfc-gh-helmeleegy) [reply tox.ini:7]: Got it.

**Review by sfc-gh-helmeleegy — APPROVED** [review]
LGTM, thanks for taking carte of this!

**Review by sfc-gh-nkumar — APPROVED** [review]

**Pull request merged**